### ISSUE TITLE
The modal component is difficult to use at 400% zoom and 1280 px width (violation)

### ISSUE THREAD

**Issue by Maren-SSB** [PxTools/pxtools-project#15]
https://statistics-norway.atlassian.net/browse/PXWEB2-743?atlOrigin=eyJpIjoiODkwNDc2ZjdhYTg3NDk1MjkzMmU2YzZlYTMzZWUzMDQiLCJwIjoiaiJ9
